ft_prepare_sourcemodel.m

```diff
@@ -706,25 +706,21 @@
   end
 end
 
-if ~isempty(cfg.movetocentroid) && strcmp(cfg.movetocentroid, 'yes')
-    % 1. compute centroids
+if strcmp(cfg.movetocentroid, 'yes')
```

Comment:
if I now run:
```
cfg                 = [];
cfg.resolution      = 1; 
cfg.method          ='basedonresolution';
cfg.headmodel       = mesh;
cfg.headmodel.type  = 'simbio';
reg_grid = ft_prepare_sourcemodel(cfg);
```
then I get the error:
Reference to non-existent field 'movetocentroid'.

Reply:
the is an "s" missing on line 710 where strcmp(..., 'yes') is checked. I will fix it. I will also add the option to the documentation....

See commit 266e2aec5cfadfc102d9873125f7ee5960a644f9